AUTHORS

```diff
@@ -5,3 +5,4 @@
 
 Francisco Bernal <fbernaly@gmail.com>
 Bharat Biradar <biradar.1.bharat@gmail.com>
+Om Phatak <everythingoutdated@gmail.com>
```

Comment:
Thanks for your contributions. Once this PR is merged, you will be listed as a contributor as many others have done. This file is only for those who have written a plug-in from scratch: a whole new ML kit feature/API from scratch. Please, revert the changes in this file. Thanks.